astroscrappy/utils/setup_package.py

```diff
@@ -1,39 +1,36 @@
 import os
 import numpy as np
 
-from distutils.core import Extension
+from setuptools import Extension
 from extension_helpers import add_openmp_flags_if_available
 
 UTIL_DIR = os.path.relpath(os.path.dirname(__file__))
 
 
 def get_extensions():
 
-    med_sources = [str(os.path.join(UTIL_DIR, "median_utils.pyx")),
-                   str(os.path.join(UTIL_DIR, "medutils.c"))]
+    med_sources = [os.path.join(UTIL_DIR, "median_utils.pyx"),
+                   os.path.join(UTIL_DIR, "medutils.c")]
 
-    im_sources = [str(os.path.join(UTIL_DIR, "image_utils.pyx")),
-                  str(os.path.join(UTIL_DIR, "imutils.c"))]
+    im_sources = [os.path.join(UTIL_DIR, "image_utils.pyx"),
+                  os.path.join(UTIL_DIR, "imutils.c")]
 
     include_dirs = [np.get_include(), UTIL_DIR]
-    libraries = []
 
     if 'CFLAGS' in os.environ:
         extra_compile_args = os.environ['CFLAGS'].split()
     else:
         extra_compile_args = ['-g', '-O3', '-funroll-loops', '-ffast-math']
```

Comment:
@cmccully - Not sure if we should have those flags by default, especially `-ffast-math` which is an option that people should use carefully (https://simonbyrne.github.io/notes/fastmath/)